src/views/forms/ClosePositionForm.tsx

```diff
@@ -82,8 +83,9 @@ export const ClosePositionForm = ({
   const { stepSizeDecimals } = useSelector(getCurrentMarketConfig, shallowEqual) || {};
   const { size: sizeData, summary } = useSelector(getInputClosePositionData, shallowEqual) || {};
   const { size, percent } = sizeData || {};
-  const hasUncommittedOrders = useSelector(calculateHasUncommittedOrders);
   const currentInput = useSelector(getCurrentInput);
+  const latestOrderStatus = useSelector(getLatestOrderStatus) ?? '';
+  const latestOrderId = useSelector(getLatestOrderId);
```

Comment:
nit: is `latestOrderId` needed? It seems we only depend on `latestOrderStatus`

Reply:
without `latestOrderId` as a value in the dep array, it is possible for the side effect to be skipped:
- the `latestOrderStatus` may not change between 2 different orders